meteor/src/lib/MochaRunner.coffee

```diff
@@ -87,10 +88,11 @@ class practical.MochaRunner
       log.return()
 
 
+@MochaRunner = practical.MochaRunner.get()
+
 if Meteor.isClient
 # Run the tests on Meteor.startup, after all code is loaded and ready
   Meteor.startup ->
-    practical.MochaRunner.get().runEverywhere()
-else
-# Run the ctor, so publication will be published
-  practical.MochaRunner.get()
+    # Defer to wait for custom reporter setup
```

Comment:
We defer because if another package set a different reporter on Meteor.startup, that's the reporter that we want to be used.